ui/src/utils/axios.js

```diff
@@ -108,11 +108,11 @@ export default (callback, store, router) => {
 
             // Authentication expired
             if (errorResponse.response.status === 401 &&
-                store.getters["auth/isLogged"]) {
+                store.getters["auth/isLogged"] &&
+                !document.cookie.split("; ").map(cookie => cookie.split("=")[0]).includes("JWT")) {
```

Comment:
Isnt there a way to get cookie date creation or something like that in it ?

Reply:
Nope, see [this](https://stackoverflow.com/a/3274904).
However this thing works as expected since the cookie will be automatically erased after expiration was achieved.
The only thing missing is not to have to do an API call to know that the cookie has expired, but for that you need a listener or something as explained [here](https://github.com/kestra-io/kestra-ee/issues/206)